public/index.html

```diff
@@ -288,10 +288,10 @@ <h5 class="widgetSubTitle" style="margin-bottom: 20px;">Customization Options (O
 							placeholder="Forest Name"><br />
 						<select class="form-select" onchange="changeAttribute('treeTenantCounter',this.value,'theme')"
 							aria-label="Default select example">
-							<option value="light" selected>Select Theme</option>
+							<option value="light" >Select Theme</option>
 							<option value="light">Light</option>
 							<option value="dark">Dark</option>
-							<option value="forest">Forest</option>
+							<option value="forest" selected>Forest</option>
```

Comment:
This does not work. The light theme is still shown when first changing to the Tree-Tenant-Counter Widget for me.